.gitignore

```diff
@@ -36,8 +36,10 @@ dist
 
 # workspace specific
 site/blocks-data.json
+site/playwright-report
+site/test-results
 site/public/blocks
 site/public/schemas
 site/site-map.json
 
-.parcel-cache
+var
```

Comment:
`var` stores service logs during integration tests. I added it to `.gitignore` in case if someone wants to run the commands from the YAML locally. We might also use this local folder for some other stuff.